loadtest/user/userentity/actions.go

```diff
@@ -127,3 +127,8 @@ func (ue *UserEntity) GetChannelUnread(channelId string) (*model.ChannelUnread,
 	return channelUnreadResponse, resp.Error
 
 }
+
+func (ue *UserEntity) GetUsersStatusesByIds(userIds []string) ([]*model.Status, error) {
```

Comment:
Is there a point to check here for user initialization like [here](https://github.com/mattermost/mattermost-load-test-ng/pull/17/files#diff-b68f89ec953f6d29ef590768f6d33f9eR124) for example? I understand that this should be done if we're going to use the user object but here I don't see the point